platform_versioned_docs/version-24.3/troubleshooting_and_faqs/nextflow.mdx

```diff
@@ -211,3 +211,18 @@ Each Seqera Platform release uses a specific nf-launcher image by default. This
 
 - Use a [pre-run script](../launch/advanced.mdx#pre--post-run-scripts) to set the desired Nextflow version. For example: `export NXF_VER=22.08.0-edge`
 - For jobs executing in an AWS Batch compute environment, create a [custom job definition](../../version-24.2/enterprise/advanced-topics/custom-launch-container.mdx) which references a different nf-launcher image.
+
+## Spot Instance failures and retries in Nextflow
+
+Up to version 24.10, Nextflow silently retried Spot Instance failures up to five times when using AWS Batch or Google Batch. These retries were controlled by cloud-specific configuration parameters (e.g., `aws.batch.maxSpotAttempts`) and happened in cloud infrastructure without explicit visibility to Nextflow. 
+
+From version 24.10, the default Spot reclamation retry setting changed to `0` on AWS and Google. By default, no _internal_ retries are attempted on these platforms. Spot reclamations now lead to an immediate failure, exposed to Nextflow in the same way as other generic failures (returning for example, `exit code 1` on AWS). Nextflow will treat these failures like any other job failure unless you actively configure a retry strategy.
```

Comment:
```suggestion
From version 24.10, the default Spot reclamation retry setting changed to `0` on AWS and Google. By default, no _internal_ retries are attempted on these platforms. Spot reclamations now lead to an immediate failure, exposed to Nextflow in the same way as other generic failures (returning, for example, `exit code 1` on AWS). Nextflow will treat these failures like any other job failure unless you actively configure a retry strategy.
```

Reply:
Note that `0` and `exit code 1` were already coded here. So if you disagree with the above, these should be made consistent.